lib/ansible/cli/galaxy.py

```diff
@@ -1396,8 +1397,10 @@ def execute_list_collection(self, artifacts_manager=None):
 
         output_format = context.CLIARGS['output_format']
         collections_search_paths = set(context.CLIARGS['collections_path'])
+        if context.CLIARGS['basedir']:
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example. Also, add a test for this change.

Reply:
I believe the changelog fragment is done, will work on the test now

Reply:
@Akasurde tests may take a while =D